ocs_ci/ocs/ocp.py

```diff
@@ -310,7 +310,10 @@ def patch(self, resource_name='', params=None, type='json'):
         """
         resource_name = resource_name or self.resource_name
         params = "\'" + f"{params}" + "\'"
-        command = f"patch {self.kind} {resource_name} -n {self.namespace} -p {params} --type {type}"
+        if type:
```

Comment:
type is a reserved keyword in Python and it might cause issues.
Can you change it to 'format_type'?

Reply:
Changed to format_type